src/Tools/Source/CompilerGeneratorTools/Source/VisualBasicSyntaxGenerator/GreenNodes/GreenNodeWriter.vb

```diff
@@ -442,6 +442,8 @@ Friend Class GreenNodeWriter
         Select Case type
             Case "Integer"
                 Return "ReadInt32"
+            Case "SyntaxKind"
```

Comment:
No, only VB.  In fact XmlNameTokenSyntax  is the only node that has an enum that needs to be serialized.